runt.toml

```diff
@@ -32,3 +32,15 @@ expect_dir = "tests/integration-verilog"
  ./bin/gen-vcd {} 2>/dev/null | vcdump | jq "$(cat {}.jq)"
  """
  expect_dir = "tests/integration-simulation"
+
+[[tests]]
+name = "polybench"
+paths = [ "benchmarks/*.fuse" ]
+cmd = """
+full=$(basename {})
+f=${full%.*}
+make -C benchmarks $f.futil-out
+cat benchmarks/$f.futil-out
+rm benchmarks/$f.futil-out
```

Comment:
Remember to make this an actual script in bin/

Reply:
yes, I'm debugging rn